### PR TITLE
AUI-3139 Aligning the Scheduler tables verticals

### PR DESCRIPTION
**Pull request by interaminense**

### CODE REVIEW

**Comment by alinedoleron** [comment]
LGTM

File: src/aui-scheduler/js/aui-scheduler-view-day.js [hunk 1]
```diff
@@ -1044,15 +1044,19 @@ var SchedulerDayView = A.Component.create({
 
                 headerView.plotEvents();
 
-                instance.headerNode.setStyle('paddingRight', getScrollbarWidth());
```
Comment (brunobasto) [inline src/aui-scheduler/js/aui-scheduler-view-day.js]: @interaminense Can you check if the function `getScrollbarWidth` is still needed?

Comment (jonmak08) [reply src/aui-scheduler/js/aui-scheduler-view-day.js]: Hey @brunobasto,

Yup that can be removed now.  I am reviewing this pull and will merge this in today.

Please let me know if you have any questions. Thanks!

**Comment by jonmak08** [comment]
Hey @interaminense and @alinedoleron,

I am reviewing this pull request and noticed that the table headers are still misaligned.

image

Is that correct or should that be fixed as well?

Please let me know if you have any questions. Thanks!

**Comment by interaminense** [comment]
Hey @jonmak08,
Is that should that be fixed as well! I'm going to solve this. 

Thank you!

**Comment by jonmak08** [comment]
Hey @interaminense and @alinedoleron,

I am going to close this pull request for now since that header needs to be fixed.  Feel free to send me another pull request or reopen this one when that issue is also fixed.

Please let me know if you have any questions. Thanks!

**Pull request closed without merging**